corehq/apps/domain/forms.py

```diff
@@ -609,15 +616,19 @@ def __init__(self, *args, **kwargs):
         self.helper[0] = twbscrispy.PrependedText('restrict_superusers', '')
         self.helper[1] = twbscrispy.PrependedText('secure_submissions', '')
         self.helper[2] = twbscrispy.PrependedText('secure_sessions', '')
-        self.helper[3] = twbscrispy.PrependedText('allow_domain_requests', '')
-        self.helper[4] = twbscrispy.PrependedText('hipaa_compliant', '')
-        self.helper[5] = twbscrispy.PrependedText('two_factor_auth', '')
-        self.helper[6] = twbscrispy.PrependedText('strong_mobile_passwords', '')
+        self.helper[3] = crispy.Field('secure_sessions_timeout')
+        self.helper[4] = twbscrispy.PrependedText('allow_domain_requests', '')
+        self.helper[5] = twbscrispy.PrependedText('hipaa_compliant', '')
+        self.helper[6] = twbscrispy.PrependedText('two_factor_auth', '')
+        self.helper[7] = twbscrispy.PrependedText('strong_mobile_passwords', '')
```

Comment:
not blocking but man this number-based indexing makes this file annoying to maintain.

Reply:
Totally agree, I just couldn't find another way to do it. It looks like crispy [used to support an add_input function](https://stackoverflow.com/questions/16581057/formhelper-object-has-no-attribute-append-while-using-crispy-forms-in-django) but it doesn't work now; I don't see anything similar in the [current docs](https://django-crispy-forms.readthedocs.io/en/latest/form_helper.html).